conf/messages

```diff
@@ -211,9 +211,10 @@ goodsVatRate.Twenty = 20% VAT
 goodsVatRate.error.required = Select which VAT rate applies to the goods
 
 # SearchGoodsCountry
-searchGoodsCountry.title = In what country did you buy the {0}?
-searchGoodsCountry.heading = In what country did you buy the {0}?
-searchGoodsCountry.hint = If you bought the goods on a plane or boat, enter the country you were travelling from at the time of purchase.
```

Comment:
you need to add this hint back in as you are using it again if declarationType == Import